src/preamble.js

```diff
@@ -2032,4 +2032,343 @@ Module['FS_createPreloadedFile'] = FS.createPreloadedFile;
 var cyberDWARFFile = '{{{ BUNDLED_CD_DEBUG_FILE }}}';
 #endif
 
+#if BINARYEN
+function integrateWasmJS(Module) {
```

Comment:
Yeah, the location changed a little, but it should not matter, it's just a function declaration, and the call just needs to happen before we try to access the asm/wasm module (so doing it in preamble is ok, but postamble would have been wrong).